lib/Brass/Actions.pm

```diff
@@ -0,0 +1,25 @@
+package Brass::Actions;
+
+use strict;
+use warnings;
+
+{
+    my @allowed_action = qw(
+        admonitor
+        backups-gpg
+        backups-s3
+        system
+        wazuh
+    );
+    sub allowed_actions {
+        return @allowed_action;
+    }
+
+    my %is_allowed_action = map { $_ => undef } @allowed_action;
+    sub is_allowed_action {
+        my $action_name = $_[0];
+        return exists $is_allowed_action{$action_name};
+    }
+
+}
```

Comment:
Thanks Tom. Is there a reason for all this being within its own block?

Reply:
Before I moved this to its own module, it was in `Brass::API` and I wanted to keep the variables private.  There's no good reason for it here, though, so I'll remove the block.